### DISCUSSION TITLE
Migrating github actions checkout version to 3

### DISCUSSION THREAD

**Discussion by andy5995** [widelands/widelands#5416] [General]
The default appears to be v3 now

https://github.com/actions/checkout

It might be good to try changing a few of the instances in build.yml and see what happens (probably after the next WL release).

It's a matter of changing `uses: actions/checkout@v2` to `uses: actions/checkout@v3`

**Comment by Noordfrees** [comment]
Since this is used only by the CI, no harm in trying out the change right away. I added it to draft PR #5415.
